packages/core/src/http-client/http-client.ts

```diff
@@ -207,6 +206,8 @@ function mergeRequestWithAxiosDefaults(request: HttpRequest): HttpRequest {
 }
 
 function executeWithAxios(request: HttpRequest): Promise<HttpResponse> {
+  // https://example.com/sap/opu/odata/sap/API_TEST_SRV/TestActionImportNoParameterNoReturnType?$format=json
+  // https://example.com/sap/opu/odata/sap/API_TEST_SRV/TestActionImportNoParameterNoReturnType?$format=json
```

Comment:
Is this left in here intentionally? Why the same URL twice?

Reply:
No, I also noticed that, it's gone.